### PR TITLE
Python Provider GA Update Samples

### PR DESCRIPTION
**Pull request by mrm9084**
Updating to GA version
Fixing keyvault usage

### CODE REVIEW

File: examples/Python/python-django-webapp-sample/README.md [hunk 1]
```diff
@@ -62,10 +62,13 @@ export AZURE_APPCONFIG_ENDPOINT="your-store-enpoint"
 
 Start the flask application using the flask command:
```
Comment (albertofori) [inline examples/Python/python-django-webapp-sample/README.md:63]: Oversight from my previous PR. "Start the **_django_** application:"

File: examples/Python/python-django-webapp-sample/README.md [hunk 2]
```diff
@@ -60,12 +60,15 @@ $Env:AZURE_APPCONFIG_ENDPOINT="your-store-endpoint"
 export AZURE_APPCONFIG_ENDPOINT="your-store-enpoint"
 ```
 
-Start the flask application using the flask command:
+Start the flask application using the django command:
```
Comment (albertofori) [inline examples/Python/python-django-webapp-sample/README.md:63]:
```suggestion
Start the django application using the following command:
```

**Review by albertofori — APPROVED** [review]

File: examples/Python/python-django-webapp-sample/quickstartproject/settings.py [hunk 3]
```diff
@@ -24,11 +24,12 @@
...
-                       key_vault_options=keyvault_options,
-                       credential=credential,
-                       selects=[selects],
-                       trimmed_key_prefixes=["testapp_settings_"])
+selects_secret = SettingSelector(key_filter="secret_key")
+config = load(endpoint=ENDPOINT,
+                                 key_vault_options=keyvault_options,
+                                 credential=credential,
+                                 selects=[selects, selects_secret],
+                                 trim_prefixes=["testapp_settings_"])
```
Comment (zhenlan) [inline examples/Python/python-django-webapp-sample/quickstartproject/settings.py:32]: NIT: The indentation doesn't align with the endpoint.

File: examples/Python/python-flask-webapp-sample/app.py [hunk 4]
```diff
@@ -14,12 +14,13 @@
...
+selects_secret = SettingSelector(key_filter="secret_key")
 keyvault_options = AzureAppConfigurationKeyVaultOptions(credential=credential)
-azure_app_config = load_provider(endpoint=ENDPOINT,
+azure_app_config = load(endpoint=ENDPOINT,
                                  key_vault_options=keyvault_options,
                                  credential=credential,
-                                 selects=[selects],
-                                 trimmed_key_prefixes=["testapp_settings_"])
+                                 selects=[selects, selects_secret],
+                                 trim_prefixes=["testapp_settings_"])
```
Comment (zhenlan) [inline examples/Python/python-flask-webapp-sample/app.py:23]: NIT: align the indentation

**Pull request merged**